api/buckets/service.go

```diff
@@ -553,7 +553,13 @@ func (s *Service) RemovePath(ctx context.Context, req *pb.RemovePathRequest) (*p
 		return nil, err
 	}
 	if err = s.IPFSClient.Pin().Update(ctx, buckPath, dirpth); err != nil {
-		return nil, err
+		if err.Error() == "'from' cid was not recursively pinned already" {
```

Comment:
Ouch! :(

Reply:
No big deal, this was only reachable by encountering bucket save errors from a schema mismatch... so, shouldn't have happened to anyone here.